pages/index.tsx

```diff
@@ -18,7 +19,7 @@ export default function Home() {
       </Head>
 
       <main>
-        <NavBar />
+        <NavBar backgroundColor={Grey010} />
```

Comment:
it's not a change, but we got two flavors of Navbars here, and the color is the only difference. It was of this color before on the index page though, so what's changed is the fact that there is this prop. Anyway, I guess this could be a default color too.